app/models/box.rb

```diff
@@ -19,7 +19,8 @@ class Box < ApplicationRecord
   include Colorized
 
   belongs_to :tenant
-  belongs_to :api_connection
+  has_many :boxes_api_connections
+  has_many :api_connections, through: :boxes_api_connections
   has_many :boxes_other_api_connections, dependent: :destroy
   has_many :other_api_connections, through: :boxes_other_api_connections, class_name: 'ApiConnection', source: :api_connection
```

Comment:
Chápem správne, že `:boxes_other_api_connections` a `:other_api_connections` chceme zmazať? Ak áno, preoč ich nezmažmeme teraz?

Reply:
Mohli by sme aj spolahnut na to, ze to tam jedno zbehnutie boxify metody doplni, ale radsej by som si nechala aj tieto povodne asociacie, minimalne kvoli kontrole/rychlemu rollbacku, ak by nastala taka situacia.